backend/capellacollab/projects/crud.py

```diff
@@ -29,9 +29,9 @@ def update_description(db: Session, name: str, description: str) -> DatabaseProj
     return project
 
 
-def create_project(db: Session, name: str) -> DatabaseProject:
+def create_project(db: Session, name: str, description: str | None) -> DatabaseProject:
```

Comment:
I would propose to write `description: str | None = None` to  have no conflict with `initialize_default_repository` in backend\capellacollab\core\database\__main__.py. Otherwise, why we don't simply write `description: str = ''` and adapt the corresponding project table?